src/components/Participants.jsx

```diff
@@ -1,22 +1,23 @@
-import { useParams } from "react-router";
 import { colourClassArray } from "../constants/colours";
-import { useGetUsersInSession } from "../queries/users/users-query";
-
-const Participants = () => {
-  const { roomId } = useParams();
-  const { data } = useGetUsersInSession(roomId);
-  const participants = data?.rows ?? [];
+import { avatarArray } from "../constants/avatars";
 
+const Participants = ({ participants }) => {
   return participants.map((participant, index) => {
     const colourIndex = index % colourClassArray.length;
     const colour = colourClassArray[colourIndex];
-
+    const thumbImage = avatarArray[colourIndex];
```

Comment:
The color array is longer than the avatar array. You'll get an undefined thumb image unless you mod the index by the avatarArray length:

```
const thumbImage = avatarArray[index % avatarArray.length]
```

Reply:
haha yes, totally realize this.  was just lazy about getting images.  thanks!